docker-compose.swarm.minimal.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.4'
```

Comment:
Do you think it would be possible to test this config in Travis?

Reply:
I dont really have any experience with Travis at all, so I dont really know, sorry.

Swarm is close to the same. The "deploy" part is not available in normal non-swarm docker. Docker configs also isnt available in normal docker, afaik. I have this config running on my swarm cluster, but I have no idea if/how travis can test it. Id have to look into it later.

Reply:
Ok, so we should mention this configuration in the documentation but put a warning saying this configuration is not automatically tested.

Reply:
Yes, compose will throw the following:
WARNING: Some services (authelia) use the 'deploy' key, which will be ignored. Compose does not support 'deploy' configuration - use `docker stack deploy` to deploy to a swarm.
WARNING: Some services (authelia) use the 'configs' key, which will be ignored. Compose does not support 'configs' configuration - use `docker stack deploy` to deploy to a swarm.

Sadly, I dont cant find a linter for stack configs. (Portainer has a helpful linter built in, which is what I use)

Reply:
ok but anyway if we can use it with compose it would be a good idea at some point to test it. Not a blocker for this PR though.